### DISCUSSION TITLE
Is it possible to add a osd RX/TX box in wfb-cli?

### DISCUSSION THREAD

**Discussion by lida2003** [svpcom/wfb-ng#395] [Questions] [answered]
Currently, it's 14560/14550 for default mavlink RX/TX, which can be saw in `wfb-cli`.

Is it possible to add a custom debug link RX/TX box in wfb-cli?

Or simple change gs mavlink to debug link, which might be 14460/14450?

**Comment by svpcom** [comment, marked as answer]
@lida2003 All streams served by wfb-ng will be automatically displayed in wfb-cli
If you need additional stream then just define it in /etc/wifibroadcast.cfg (see master.cfg for reference and examples)

**Reply by lida2003** [reply]
Oh, I have used console command `wfb_tx -p 17 -u 14560 -K /etc/drone.key -B 20 -M 1 -S 1 -L 1 -G long -k 1 -n 2 -T 0 -i 7669206 -f data wlan0 `. I'll check how to write the configure in `/etc/wifibroadcast.cfg`.
